ci-operator/config/openshift/lightspeed-operator/openshift-lightspeed-operator-main.yaml

```diff
@@ -107,7 +114,53 @@ tests:
       resources:
         requests:
           cpu: 100m
-    workflow: hypershift-hostedcluster-workflow
+    workflow: generic-claim
+- as: bundle-e2e-4-16
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-east-1
+    owner: ols
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.16"
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    pre:
+    - ref: ipi-install-rbac
+    test:
+    - as: e2e-install
+      cli: latest
+      commands: |
+        oc create namespace openshift-lightspeed
```

Comment:
let's label the namespace for in-cluster monitoring
```
oc label namespaces openshift-lightspeed openshift.io/cluster-monitoring=true --overwrite=true
```

Reply:
openshift in-cluster monitoring requires this label to scrape metrics from that namespace. 
this will fix the test.